example/src/App.tsx

```diff
@@ -78,7 +77,7 @@ const App = () => {
       />
       {enablePolymerEditor && <PolymerToggler toggle={setShowPolymerEditor} />}
       {hasError && (
-        <ErrorModal
+        <InfoModal
```

Comment:
Good job on moving `ErrorModal` into `ketcher-react/` and good naming. However, I'd suggest using `<InfoModal />` somewhere in `ketcher-react/` instead of `example/`. Other parts are great, and the business requirement is fulfilled, good job!